studio/pages/integrations/vercel/[slug]/marketplace/choose-project.tsx

```diff
@@ -58,7 +60,13 @@ const VercelIntegration: NextPageWithLayout = () => {
   const supabaseProjects = useMemo(
     () =>
       supabaseProjectsData
-        ?.filter((project) => project.organization_id === organization?.id)
+        ?.filter(
+          (project) =>
+            project.organization_id === organization?.id &&
+            (project.status === PROJECT_STATUS['ACTIVE_HEALTHY'] ||
```

Comment:
do we need to consider UPGRADING status too?

also can be written like this for better readability
`const allowedProjectStatuses = [PROJECT_STATUS['ACTIVE_HEALTHY'], PROJECT_STATUS['COMING_UP'], ...]`

and then `allowedProjectStatuses.includes(project.status)`

Reply:
do in another PR?

Reply:
yeap thats fine - but do we need to consider UPGRADING status too?

Reply:
kept it simple.